packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -327,6 +352,78 @@ class SyncWatcher extends BaseWatcher {
       .then(async () => await this.availableLiquidityWatcher.syncBonderCredit())
   }
 
+  async handleTransferSentToL2Event (event: TransferSentToL2Event) {
+    const {
+      chainId: destinationChainIdBn,
+      recipient,
+      amount,
+      amountOutMin,
+      deadline,
+      relayer,
+      relayerFee
+    } = event.args
+    const { transactionHash, logIndex } = event
+    const transferId = getTransferSentToL2TransferId(
+      Number(destinationChainIdBn.toString()),
+      recipient,
+      amount,
+      amountOutMin,
+      deadline,
+      relayer,
+      relayerFee,
+      transactionHash,
+      logIndex
+    )
+    const logger = this.logger.create({ id: transferId })
+    logger.debug('handling TransferSentToL2 event')
+
+    try {
+      const blockNumber: number = event.blockNumber
+      const l1Bridge = this.bridge as L1Bridge
+      const sourceChainId = await l1Bridge.getChainId()
+      const destinationChainId = Number(destinationChainIdBn.toString())
```

Comment:
since these two do the same casting

```ts
      Number(destinationChainIdBn.toString()),
```

```ts
      const destinationChainId = Number(destinationChainIdBn.toString())
```

the const destinationChainId line can be moved to the top to make it more dry